.gitignore

```diff
@@ -41,3 +41,4 @@ CMakeLists.txt.user
 /.idea/
 # Ignore Visual Studio Code's working dir
 /.vscode/
+/Source/UnitTests/Sys
```

Comment:
This screams at me that you're doing something wrong. A build should never affect the source directory.

Reply:
I don't particularly like this either but I haven't figured out what's going wrong with how I'm doing this and haven't gotten any responses when I've asked about it. The weird problem is that when the two Windows autobuilds build and run this unit test, they run the following
```  std::string cur_directory = File::GetCurrentDir();
  std::string sys_directory = cur_directory + DIR_SEP + "Sys";
  const auto& list_filepath = fmt::format("{}{}{}", sys_directory, DIR_SEP, APPROVED_LIST_FILENAME);
  ASSERT_TRUE(JsonFromFile(list_filepath, &json_tree, &error))
      << "Failed to open file at " << list_filepath;```
and the resulting error I get is that list_filepath is looking in _Source_/UnitTests/Sys for the file. I have no idea why the auto builds would attempt to do this and what to do to get them to go somewhere more reasonable, like Binary/x64/Release/UnitTests/Sys which is where the other autobuilds that use CMake are looking.

Reply:
Okay, so that just means the working directory is set wrong/weird for the unit test executable invokation. Let's see if I can figure out where that is set...

Reply:
Don't forget to remove this now that we have a better solution.